src/lib/sparse_ledger_lib/sparse_ledger.ml

```diff
@@ -213,42 +291,59 @@ let%test_module "sparse-ledger-test" =
       let data_hash t = Md5.digest_string (Binable.to_string (module T) t)
     end
 
-    include Make (Hash) (String) (Account)
+    module Key = struct
+      module Stable = struct
+        module V1 = struct
+          module T = struct
+            type t = string [@@deriving bin_io, eq, sexp, version]
+          end
+
+          include T
+        end
+
+        module Latest = V1
+      end
+
+      include Stable.V1
```

Comment:
The `include` makes `Key.t` have `bin_io` and `version`, which I don't think is what we want.

I would write:

`type t = Stable.Latest.t [@@deriving eq, sexp]`